modules/measurement/budget-calculator/load.php

```diff
@@ -0,0 +1,237 @@
+<?php
+/**
+ * Module Name: Budget Calculator
+ * Description: Allows users to define their own metrics and generate a budget.json file from them.
+ * Experimental: No
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+/**
+ * Display the content of the budget calculator option page.
+ *
+ * @since n.e.x.t
+ */
+function budget_calc_page_content() {
+	?>
+	<div class="wrap">
+	<h1>Budget Calculator</h1>
+	<form method="post" action="options.php">
+	<?php
+
+	settings_fields( 'budget_calc_settings' );
+	do_settings_sections( 'budget-calculator' );
+	submit_button( __( 'Download budget.json', 'performance-lab' ) );
+
+	?>
+	</form></div>
+	<?php
+}
+
+/**
+ * Register the option page.
+ *
+ * @since n.e.x.t
+ */
+function budget_calc_menu_page() {
+	add_menu_page(
+		'Budget Calculator',
+		'Budget Calculator',
+		'manage_options',
+		'budget-calculator',
+		'budget_calc_page_content',
+		'dashicons-calculator'
+	);
+}
+add_action( 'admin_menu', 'budget_calc_menu_page' );
+
+/**
+ * Register the settings for the option page.
+ *
+ * @since n.e.x.t
+ */
+function budget_calc_register_settings() {
+	register_setting(
+		'budget_calc_settings',
+		'budget_calc_options',
+		array(
+			'sanitize_callback' => 'budget_calc_sanitize_settings',
+		)
+	);
+
+	add_settings_section( 'budget_calc_settings_id', 'Budget Calculator Metrics', '', 'budget-calculator' );
+
+	$options        = get_option( 'budget_calc_options', array() );
+	$range_settings = array(
+		'HTML'       => 300,
+		'CSS'        => 200,
+		'Font'       => 200,
+		'Images'     => 3000,
+		'JavaScript' => 750,
+	);
+
+	foreach ( $range_settings as $title => $max ) {
+		$id = strtolower( $title ) . '_range';
+
+		add_settings_field(
+			$id,
+			$title,
+			function () use ( $options, $id, $max ) {
+				budget_calc_render_range_field( $options, $id, $max );
+			},
+			'budget-calculator',
+			'budget_calc_settings_id'
+		);
+	}
+
+	add_settings_field(
+		'updated',
+		'',
+		'budget_calc_render_hidden_updated_field',
+		'budget-calculator',
+		'budget_calc_settings_id',
+		array(
+			'class' => 'hidden',
+		)
+	);
+}
+add_action( 'admin_init', 'budget_calc_register_settings' );
+
+/**
+ * Callback to sanitize the input from the settings form.
+ *
```

Comment:
`* @since n.e.x.t`

Reply:
👍 All done now

Reply:
Thank you for fix it @gabriel-detassigny